package.json

```diff
@@ -26,15 +26,15 @@
   },
   "homepage": "https://github.com/TokenScript/token-negotiator#readme",
   "dependencies": {
-    "@tokenscript/attestation": "^1.0.7-alpha",
+    "@tokenscript/attestation": "^0.3.1",
```

Comment:
The versioning scheme has been puzzling me. I noticed earlier today at https://www.npmjs.com/package/@tokenscript/attestation that the numbering had seemingly gone backwards. Is this intentional/by-design?

Reply:
Hi @samwarwick, GP. Let's keep to one approach.

It seems in this case the method of versioning has changed (from alpha .... which would then start at 1.0.0 once production ready / to 0.3.1 which will work towards 1.0.0). There are semver tools that can prompt us of the type of changes are being added e.g. minor, major / breaking changes, bug fix. There's an issue inside Git to implement this before we get to being prod ready.

Reply:
Adding the existing Token Negotiator ticket for reference: https://github.com/TokenScript/token-negotiator/issues/65. Might need something similar for Attestation.

Reply:
> The versioning scheme has been puzzling me. I noticed earlier today at https://www.npmjs.com/package/@tokenscript/attestation that the numbering had seemingly gone backwards. Is this intentional/by-design?

Hi @samwarwick this was done to keep the major/minor version in sync with the Java attestation library.